Documentation/devel/stage1-implementors-guide.md

```diff
@@ -16,36 +16,47 @@ This separation of concerns is reflected in the file-system and layout of the co
+The environment variable "RKT_LOCK_FD" contains the file descriptor number of the open directory handle for "/var/lib/rkt/pods/$uuid".
+It is necessary that stage 1 leave this file descriptor open and in its locked state for the duration of the `rkt run`.
 
-In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at "/init" within the stage 1 ACI rootfs.  The majority of its execution entails generating a systemd-nspawn argument list and writing systemd unit files for the constituent apps before executing systemd-nspawn.  Systemd-nspawn then boots the stage 1 systemd with the just-written unit files for launching the contained apps.  The "/init" program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
+In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at "/init" within the stage 1 ACI rootfs.
+The majority of its execution entails generating a systemd-nspawn argument list and writing systemd unit files for the constituent apps before executing systemd-nspawn.
+Systemd-nspawn then boots the stage 1 systemd with the just-written unit files for launching the contained apps.
+The "/init" program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
 
-An alternative stage 1 could forego systemd-nspawn and systemd altogether, or retain these and introduce something like novm or qemu-kvm for greater isolation by first starting a VM.  All that is required is an executable at the place indicated by the "coreos.com/rkt/stage1/run" entrypoint which knows how to apply the Pod Manifest and prepared ACI file-systems to good effect.
+An alternative stage 1 could forego systemd-nspawn and systemd altogether, or retain these and introduce something like novm or qemu-kvm for greater isolation by first starting a VM.
+All that is required is an executable at the place indicated by the "coreos.com/rkt/stage1/run" entrypoint which knows how to apply the Pod Manifest and prepared ACI file-systems to good effect.
 
-The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`. rkt supports two ways of doing that, stage1 implementors must pick one:
+The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`.
+rkt supports two ways of doing that, stage1 implementors must pick one:
 
 1. Writing to "/var/lib/rkt/pods/$uuid/pid" the PID of the process that is PID 1 in the container.
-2. Writing to "/var/lib/rkt/pods/$uuid/ppid" the PID of the parent of the process that is PID 1 in the container. This assumes it has only one child.
+2. Writing to "/var/lib/rkt/pods/$uuid/ppid" the PID of the parent of the process that is PID 1 in the container.
+   This assumes it has only one child.
 
 #### Arguments
```

Comment:
NB (later, jx): `rkt` is a *command*, `run` is a *subcommand*, `--debug` is an *option*, and the name of the ACI to execute is an *argument*.